scripts/generate_psa_key_attributes.py

```diff
@@ -98,11 +118,13 @@ def __init__(self,
     def pack(self):
         """Builds a binary blob compatible with the psa_key_attributes_s C struct"""
 
+        location_lifetime = self.location | self.lifetime
```

Comment:
I am not sure if it would have been better for readability to treat location and lifetime as separate bitfields. Right now the lowest byte for the location word is 0x00 (kind of pre-shifted) because we are going to 'or' in the lifetime. But I guess pack doesn't support 24-bit words, so at some point we will still need to do the 'shift' and 'or' operation to create 32-bit word that we can give to pack().

Reply:
```suggestion
        location_lifetime = self.location | (self.lifetime & 0xFF)
```
Maybe add this for convenience and/or a comment on pre-shifted `location`